management/src/clusterm/manager/api.go

```diff
@@ -72,11 +73,12 @@ func (m *Manager) apiLoop(errCh chan error) {
 	}
 }
 
-type postCallback func(tagsOrAddrs []string, sanitizedExtraVars string) error
+type postCallback func(tagsOrAddrs []string, flags ActionFlags) error
```

Comment:
@mapuri I think in that case, I will make APIRequest as the argument. and add extraVars also to it.